edalize/libero.py

```diff
@@ -186,19 +194,39 @@ def tcl_file_filter(self, f):
             return file_types[_file_type] + f.name
         return ""
 
-    def constraint_file_filter(self, f, type="ALL"):
+    def syn_constraint_file_filter(self, f):
+        file_types = {
+            "SDC": "constraint/",
+            "FDC": "constraint/",
+            "NDC": "constraint/",
+        }
+        _file_type = f.file_type.split("-")[0]
```

Comment:
What is this split for?

Reply:
From what I can tell, it might just be used to convert `file_type`s like [`vhdlSource-2008`](https://github.com/olofk/edalize/blob/main/edalize/quartus.py#L139) to just `vhdlSource`. It seems like that pattern's used in a few other toolchains, like [quartus](https://github.com/olofk/edalize/blob/main/edalize/quartus.py#L139).

Reply:
That would be my guess too, but since none of the potential matches contain dashes in the file type I suspect this whole function (as well as the other ones) can just be simplified to:
```python
    def pnr_constraint_file_filter(self, f):
        if _file_type in ["PDC", "SDC", "FPPDC"]:
            return f.name
```